lambda_function.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+import urllib3
+import boto3
+from urllib3.exceptions import ConnectTimeoutError
+from boto3 import resource
+from botocore.exceptions import ClientError
+from boto3.dynamodb.conditions import Key, Attr
+
+http = urllib3.PoolManager()
+client = boto3.client('events', region_name="eu-west-2")
```

Comment:
I'm not sure if these are needed.

If they are is it possible to remove the region_name as boto3 should read the current region from the AWS_REGION environment variable.